internal/cluster/distribution/eks/eksworkflow/common.go

```diff
@@ -40,15 +40,23 @@ type AWSSessionFactory interface {
 // getStackTags returns the tags that are placed onto CF template stacks.
 // These tags  are propagated onto the resources created by the CF template.
 // TODO: move these to a better place
-func getStackTags(clusterName, stackType string) []*cloudformation.Tag {
-	return append([]*cloudformation.Tag{
+func getStackTags(clusterName, stackType string, clusterTags map[string]string) []*cloudformation.Tag {
+	tags := append([]*cloudformation.Tag{
 		{Key: aws.String("banzaicloud-pipeline-cluster-name"), Value: aws.String(clusterName)},
 		{Key: aws.String("banzaicloud-pipeline-stack-type"), Value: aws.String(stackType)},
 	}, internalAmazon.PipelineTags()...)
+
+	for k, v := range clusterTags {
```

Comment:
Same as above: this could overwrite/result in a validation error.

Reply:
add custom tags first